products/BellHybrid/services/db/agents/QuotesQueries.hpp

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2017-2024, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#pragma once
+
+#include <Common/Types.hpp>
+
+namespace Quotes::Queries
+{
+    constexpr auto getQuotes =
```

Comment:
You can use `inline` for those, as they're declared in header file and can potentially be used in multiple different translation units

Reply:
Done